packages/ui/README.md

```diff
@@ -10,7 +10,9 @@ Additionally you have to provide `secrets/yt-interface-secret.json` file with [a
 
 ```json
 {
-  "oauthToken": "special-user-secret-token"
```

Comment:
The field is still [actual](https://github.com/ytsaurus/ytsaurus-ui/blob/main/packages/ui/src/server/components/requestsSetup.ts#L18), cluster-specific tokens are optional.

```ts
 require(ytInterfaceSecret)?.oauthToken; // Backward compatibility
```
It is better to get rid of the comment. It is not backward compatibility, it is just another approach that allows to have one common token for all clusters.